src/sensesp/net/web/base_command_handler.cpp

```
@@ -18,18 +20,38 @@ namespace sensesp {
-/// Compares the Origin header's authority (host[:port]) against the request's
-/// own Host header. The Host header reflects whatever interface the browser
-/// actually used, so this is same-origin for every access path — soft-AP IP,
-/// station IP, mDNS name, or custom DNS — without a hardcoded allowlist.
-/// Requests without an Origin header (non-browser clients) are allowed.
-///
-/// An Origin or Host that does not fit the buffer is rejected rather than
-/// treated as absent: a legitimate same-origin request to this device is always
-/// short, so an over-long header can only be a forgery attempt and must fail
-/// closed.
+// Strips an optional ":port" suffix from a Host/Origin authority.
+String strip_port(const String& authority) {
```

Cppcheck (2.21.0)

[performance] Variable 'button_' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)